### PR TITLE
[1.24] Fix app crash when network was switch to unsupported one

### LINKED ISSUE #1322: [1.24] The app is crashed when switch to an unsupported network
Related to #1270, #1317

1. Open the app
2. Connect to MM
3. Change a network to an unsupported one in the MM

**AR**: the app is crashed

**ER:** Unsupported network warnings appear

### PR DESCRIPTION
**Pull request by shoom3301**
# Summary

Fixes #1322

### CODE REVIEW

**Comment by elena-zh** [comment]
Hey @shoom3301 , the last nitpick is that it would be great to remove a token name from the URL 
image
Thanks!

**Comment by shoom3301** [comment]
@elena-zh fixed

**Review by elena-zh — APPROVED** [review]
LGTM!

File: src/cow-react/modules/trade/hooks/useTradeNavigate.ts [hunk 1]
```diff
@@ -12,22 +14,27 @@ interface UseTradeNavigateCallback {
...
   const tradeTypeInfo = useTradeTypeInfo()
+  const { chainId: currentChainId } = useWeb3React()
 
   return useCallback(
     (chainId: SupportedChainId | null | undefined, { inputCurrencyId, outputCurrencyId }: TradeCurrenciesIds) => {
       if (!tradeTypeInfo) return
 
+      const isNetworkSupported = isSupportedChainId(currentChainId)
+      // Currencies ids shouldn't be displayed in the URL when user selected unsupported network
+      const fixCurrencyId = (currencyId: string | null) => (isNetworkSupported ? currencyId || undefined : undefined)
```
Comment (alfetopito) [inline src/cow-react/modules/trade/hooks/useTradeNavigate.ts:25]: Nitpick: can this be defined outside of the hook?

**Review by alfetopito — APPROVED** [review]
Only one nitpick

File: src/cow-react/modules/trade/hooks/useTradeNavigate.ts (hunk 1 above)
Comment (shoom3301) [reply src/cow-react/modules/trade/hooks/useTradeNavigate.ts]: Fixed

**Pull request merged**